src/main/java/uk/gov/ida/verifyserviceprovider/configuration/HubMetadataConfiguration.java

```diff
@@ -150,26 +138,22 @@ private static String generateTrustStorePath(URI uri) {
                 trustStoreName = TEST_VERIFY_TRUSTSTORE_NAME;
                 break;
             default:
-                throw new RuntimeException("No trust store configured for Metadata URI " + uri);
+                throw new RuntimeException("No trust store configured for Hub Environment: " + hubEnvironment.name());
```

Comment:
These exceptions are probably not strictly necessary - if the hubEnvironment was not recognised there would have been an exception thrown trying to parse it - but similar would have been the case before with the MetadataUri enum. I have left them, as I assume they were here in case we added more valid values to the enum without updating this (and this would be a much more useful error in that case). But I'll take them out if no one thinks there is a good reason for them to be here.

Reply:
More errors is good errors in my opinion 👍